lib/src/models/network_data.dart

```diff
@@ -1,5 +1,7 @@
+import 'package:instabug_flutter/src/models/w3c_header.dart';
+
 class NetworkData {
-  const NetworkData({
+  NetworkData({
```

Comment:
Why is it not `const` anymore? 👀

Reply:
because it has a private W3CHeader object that gets initialized in the constructor